runelite-api/src/main/java/net/runelite/api/RenderOverview.java

```diff
@@ -24,8 +24,19 @@
  */
 package net.runelite.api;
 
+import java.awt.Rectangle;
+
 public interface RenderOverview
 {
+	Point getWorldMapPosition();
+
+	Rectangle getWorldMapDisplay();
+
+	float getWorldMapZoom();
+
+	//Takes a world point
+	void setWorldMapPositionTarget(int worldPointX, int worldPointY);
```

Comment:
Should this actually take a `WorldPoint`?

Reply:
The comment is referring to the parameters being in world coordinates instead of worldMap coordinates. Most functions in RenderOverview take worldMap coordinates

Reply:
Since the other functions taking worldMap coordinates aren't exposed anymore I'll remove the comment